src/ForceSecureConnection/Redirection.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\HttpMiddleware\ForceSecureConnection;
+
+/**
+ * Redirection from HTTP to HTTPS parameters.
+ */
+final class Redirection
```

Comment:
While the value object fits the purpose fine, the name is confusing. It is not "Redirection" by itself, it does not contain a host to redirect to. I'd name it something like `RedirectToPort` but without using the verb. Also... does it make sense to use status of such redirection which is not 301?

Reply:
VO indicates that redirection should occur and contains redirection parameters in the ForceSecureConnectionMiddleware context. Seems, "Redirection" name is OK: if "Redirection" are provided, the middleware will enforce the redirection.

I think, in rare cases status code may be 302.

Reply:
I don't think we need 302 in this case. That leaves us with plain nullable string.

Reply:
I don't think we need 302 in this case. That leaves us with plain nullable string.

Reply:
`RedirectOptions`?

Reply:
Done